alpha-features.md

```diff
@@ -0,0 +1,28 @@
+# Open Service Broker API Alpha Features
+
+## Table of Contents
+  - [Alpha Overview](#overview)
+  - [How alpha features are documented](#how-alpha-features-are-documented)
+  - [Current alpha features](#current-alpha-features)
+
+## Overview
+
+The features described in this document are released at an ALPHA level.  They
+are are available for use in order for the community to provide feedback and
+are not yet an official part of the spec.
+
+Alpha features may be changed or removed in response to feedback.
+
+## How Alpha features are documented
+
```

Comment:
Is it going to be difficult to actually do this here? Taking the JSON schema/parameters change as an example; there are changes to a number of responses in the specification, and having the entire specification together makes it easier for me to understand.

Is there any way we can flag things as alpha within the spec itself?